feature-libs/cart/schematics/add-cart/schema.json

```diff
@@ -20,14 +20,34 @@
       "type": "array",
       "uniqueItems": true,
       "items": {
-        "enum": ["Saved-Cart", "Quick-Order", "Import-Export"],
+        "enum": [
+          "Cart",
+          "WishList",
+          "Saved-Cart",
+          "Quick-Order",
+          "Import-Export"
+        ],
         "type": "string"
       },
-      "default": ["Saved-Cart", "Quick-Order", "Import-Export"],
+      "default": [
+        "Cart",
+        "WishList",
+        "Saved-Cart",
+        "Quick-Order",
+        "Import-Export"
+      ],
       "x-prompt": {
         "message": "Which Cart features would you like to set up?",
         "type": "list",
         "items": [
+          {
+            "value": "Cart",
```

Comment:
main-cart?

Reply:
This is displayed in the installer "menu".  I'd leave it to cart for now.